ts-tests/tests/utils.ts

```diff
@@ -0,0 +1,353 @@
+import 'mocha';
+
```

Comment:
Please use prettier to prettify your source code.
https://prettier.io/

Some configuration, refer to litentry-registrar or push-notification repo